app/models/like.rb

```diff
@@ -0,0 +1,7 @@
+class Like < ApplicationRecord
+  belongs_to :post
+
```

Comment:
- [ ] Great job defining the association in the like model! 😊 However, in the context of our Entity-Relationship Diagram (ERD), a like should not only be associated with a post but also with a user. While you've correctly defined the connection between comments and posts, it appears that the link to users is currently missing.

  - To fully align your implementation with the ERD, please make sure to establish the necessary association between likes and users as well. This will help maintain data integrity and ensure that every like can be properly attributed to the user who created it.
  
Happy coding! 👍🚀